api4/config_local.go

```diff
@@ -139,3 +140,18 @@ func localPatchConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 		mlog.Warn("Error while writing response", mlog.Err(err))
 	}
 }
+
+func localConfigReload(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Do we need this specific handler? Local mode should bypass `App.SessionHasPermissionTo` check anyway.

Reply:
There's an additional check on `RestrictSystemAdmin`. I guess we could add a `session.IsUnrestricted()` check to avoid the method repetition.